lib/matplotlib/axes/_axes.py

```diff
@@ -5747,6 +5770,13 @@ def pcolor(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
         snap : bool, default: False
             Whether to snap the mesh to pixel boundaries.
 
+       dropdata: bool, default: True
+            If True (default), and *X* and *Y* are the same size as C in their
+            respective dimensions, drop the last element of C in both
+            dimensions.  If False, *X* and *Y* are assumed to the the midpoints
```

Comment:
to **be** ~~the~~ the midpoints

Minor spelling corrections here and in other docstrings which were copied below it looks like.